.github/workflows/integration.yaml

```
@@ -12,20 +12,27 @@ jobs:
-  
\ No newline at end of file
+      
+      - name: Capture arkd logs
+        if: failure()
+        run: |
+          echo "=== Arkd Logs ==="
+          docker logs arkd
+          echo "=== End Arkd Logs ==="
+      
+      - name: Tear down
+        run: make docker-stop
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)